Docs grammar and typos check (#6064)
vuestorefront/vue-storefront: packages/core/docs/commercetools/configuration.md

}]
```

> - `useNuxtI18nConfig` - when enabled, `@vue-storefront/commercetools/nuxt` package will use `i18n` config object provided in `nuxt.config.js`. Otherwise, the `i18n` config should be passed directly to this package configuration. You can read more about it on [Internationalization](../advanced/internationalization.md) page. There are things specific to Commercetools, which you can read more about below.

Reviewer: ```suggestion
- `useNuxtI18nConfig` - when enabled, `@vue-storefront/commercetools/nuxt` package will use `i18n` config object provided in `nuxt.config.js`. Otherwise, you can pass the `i18n` config directly to this package configuration. You can read more about it on [Internationalization](../advanced/internationalization.md) page. There are things specific to commercetools, which you can read more about below.
```

}
```
::: warning Each state requires the tax rules

> A state without tax rules cannot work properly. Make sure to add the tax rules for each state in the Commercetools as described [here](https://docs.commercetools.com/merchant-center/project-settings#configuring-taxes).

Reviewer: ```suggestion
A state without tax rules cannot work properly. Make sure to add the tax rules for each state in the commercetools as described [here](https://docs.commercetools.com/merchant-center/project-settings#configuring-taxes).
```

### `api`

- `uri` - link to your Commercetools GraphQL API instance.

> - `authHost` - link to Commercetools Authentication Server. It is used to request an access token from Commercetools OAuth 2.0 service. To choose the nearest service, please visit [Commercetools hosts list](https://docs.commercetools.com/api/authorization)

Reviewer: ```suggestion
- `authHost` - link to Commercetools Authentication Server. It is used to request an access token from commercetools OAuth 2.0 service. To choose the nearest service, please visit [commercetools hosts list](https://docs.commercetools.com/api/authorization)
```
